### ISSUE TITLE
Invoke local ReferenceError

### ISSUE THREAD

**Issue by hmorales** [serverless/serverless#12848]
### Issue description

When trying to run 
`serverless invoke local -f MyLambda`
I am getting 
```
ReferenceError: fse is not defined
    at <anonymous> (/home/h/.serverless/releases/4.4.4/serverless/lib/plugins/aws/invoke-local/index.js:467:9)
    at async Promise.all (index 0)
    at async Promise.all (index 1)
    at async AwsInvokeLocal.invokeLocalDocker (/home/h/.serverless/releases/4.4.4/serverless/lib/plugins/aws/invoke-local/index.js:618:21)
    at async PluginManager.runHooks (/home/h/.serverless/releases/4.4.4/serverless/lib/classes/plugin-manager.js:680:9)
    at async PluginManager.invoke (/home/h/.serverless/releases/4.4.4/serverless/lib/classes/plugin-manager.js:719:9)
    at async PluginManager.run (/home/h/.serverless/releases/4.4.4/serverless/lib/classes/plugin-manager.js:768:7)
    at async Serverless.run (/home/h/.serverless/releases/4.4.4/serverless/lib/serverless.js:221:5)
    at async runFramework (/home/h/.serverless/releases/4.4.4/sf-core/src/lib/runners/framework.js:291:5)
    at async route (/home/h/.serverless/releases/4.4.4/sf-core/src/lib/router.js:218:10)
```

### Context

Summary Report -----------------

# Service Overview
- Serverless Framework Version: 4.4.4
- Service Config File: serverless.yml
- Service Name: intern-hub
- Service Runtime: provided.al2
- Service Stage: dev
- Service Region: us-east-1
- Command: invoke local --function StartInternFlows

## Service Path
/home/h/Projects/social-intern/intern_hub

## Error Message
fse is not defined

## Error Stacktrace
```
at file:///home/h/.serverless/releases/4.4.4/package/dist/sf-core.js:907:17437
at async Promise.all (index 0)
at async Promise.all (index 1)
at async AwsInvokeLocal.invokeLocalDocker (file:///home/h/.serverless/releases/4.4.4/package/dist/sf-core.js:908:2587)
at async PluginManager.runHooks (file:///home/h/.serverless/releases/4.4.4/package/dist/sf-core.js:915:9311)
at async PluginManager.invoke (file:///home/h/.serverless/releases/4.4.4/package/dist/sf-core.js:915:10082)
at async PluginManager.run (file:///home/h/.serverless/releases/4.4.4/package/dist/sf-core.js:915:10813)
at async Serverless.run (file:///home/h/.serverless/releases/4.4.4/package/dist/sf-core.js:922:10612)
at async runFramework (file:///home/h/.serverless/releases/4.4.4/package/dist/sf-core.js:945:2696)
at async route (file:///home/h/.serverless/releases/4.4.4/package/dist/sf-core.js:973:13954)
```

## Service Config
```
service: intern-hub
provider:
  name: aws
  region: us-east-1
  runtime: provided.al2
  memorySize: 1024
  timeout: 30
  # environment:
  #   DB_HOST: ${env:DB_HOST}
  #   DB_NAME: ${env:DB_NAME}
  #   DB_USER: ${env:DB_USER}
  #   DB_PASS: ${env:DB_PASS}
plugins:
  - ./vendor/bref/bref
functions:
  StartInternFlows:
    handler: src/Component/StartInternFlows.php
    layers:
      - ${bref:layer.php-83}
    events:
      - schedule: rate(5 minutes)
    memorySize: 1024
    timeout: 30
    environment:
      DB_HOST: ${env:DB_HOST}
      DB_NAME: ${env:DB_NAME}
      DB_USER: ${env:DB_USER}
      DB_PASS: ${env:DB_PASS}
package:
  patterns:
    - '!node_modules/**'
    - '!tests/**'
    - '!.git/**'
    - 'vendor/**'
# Commands for local testing and deployment:
# serverless invoke local --function StartInternFlows
# serverless deploy
```

**Comment by hmorales** [comment]
Solution is to change import 
`import { ensureDir } from 'fs-extra'`
to 
`import fse from 'fs-extra'`
and update references to ensureDir to use fse.

**Comment by skierkowski** [comment]
@hmorales thanks for sharing the solution.

**Comment by hmorales** [comment]
@skierkowski looks like I incorrectly assumed that this would be fixed in an upcoming release. Still seeing the same issue in 4.4.6. Any idea when this will be resolved?

**Comment by skierkowski** [comment]
@hmorales Sorry, I misunderstood the issue. Re-opening. We'll have a look at this shortly.

**Comment by czubocha** [comment]
Hey @hmorales, the fix will be included in the next release.

**Comment by czubocha** [comment]
Hey @hmorales, the fix has been released in version [4.4.7](https://github.com/serverless/serverless/releases/tag/v4.4.7). Feel free to let us know if you encounter any further issues.
